setup.py

```diff
@@ -142,7 +142,7 @@
         'SLIST = obspy.io.ascii.core',
         'PICKLE = obspy.core.stream',
         'CSS = obspy.io.css.core',
-        'DATAMARK = obspy.io.datamark.core',
+        'WIN = obspy.io.win.core',
```

Comment:
or just a dummy obspy.io.datamark.core that answers it not longer exists and returns /dies. not a clean deprecation... but yah...